packages/maskbook/src/plugins/Trader/UI/trending/TrendingCard.tsx

```diff
@@ -0,0 +1,40 @@
+import { Card, createStyles, makeStyles } from '@material-ui/core'
+import React from 'react'
+import { useStylesExtends } from '../../../../components/custom-ui-helper'
+import { getActivatedUI } from '../../../../social-network/ui'
+
+const useStyles = makeStyles((theme) => {
+    const internalName = getActivatedUI()?.internalName
+    return createStyles({
+        root: {
+            width: 450,
+            overflow: 'auto',
+            '&::-webkit-scrollbar': {
+                display: 'none',
+            },
+            ...(internalName === 'twitter'
+                ? {
+                      boxShadow: `${
+                          theme.palette.type === 'dark'
```

Comment:
Hmm why you'd like to define this

Reply:
For different shadows in light and dark mode.

Reply:
I mean why you're specifying shadow manually? It can be fetched from the theme.